src/test/java/seedu/cafectrl/command/ListMenuCommandTest.java

```diff
@@ -7,13 +7,16 @@
 
 import java.util.ArrayList;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
 class ListMenuCommandTest {
     @Test
-    public static void execute_expectTwoDishes() {
+    public void execute_addTwoDishes_expectTwoDishes() {
         ArrayList<Dish> menuItems = new ArrayList<>();
         menuItems.add(new Dish("Chicken Rice", 2.50F));
```

Comment:
Perhaps you should create a dish object and then pass the dish objects into menuItems.add